packages/main/src/plugin/index.ts

```diff
@@ -120,15 +120,40 @@ export class PluginSystem {
     });
   }
 
+  // Create an Error to access stack trace
+  // Match a regex for the file name and return it
+  // return nothing empty string if file name not found
+  getExtName(){
+    var stack: String = '';
+
+    try{
+      throw new Error('Testing Logging');
+    }
+    catch(error){
+      if(error instanceof Error)
+      stack = error.stack || '';
+    }
+    
+    if(stack!=''){
+      var extensionGroupings = stack.match(/(podman\-desktop\\extensions)\\(lima|crc|docker|kube\-context|podman)\\(dist\\extension\.js)+:/)
```

Comment:
note that it could be other JavaScript object than the entrypoint and it's not forced to be named extension.js 

So here probably we should just check if parent folder is part of an extension